packages/palette/src/elements/Dropdown/Dropdown.tsx

```diff
@@ -132,13 +142,24 @@ export const Dropdown: React.FC<DropdownProps> = ({
       }
     }
 
+    const handleClick = (event: MouseEvent) => {
+      if (!panelRef.current || !openDropdownByClick) return
+
+      if (panelRef.current.contains(event.target as Element)) {
+        onHide()
+      }
+    }
+
     document.addEventListener("keydown", handleKeyDown)
     document.addEventListener("keyup", handleKeyUp)
+    document.addEventListener("click", handleClick)
 
     return () => {
       document.removeEventListener("keydown", handleKeyDown)
       document.removeEventListener("keyup", handleKeyUp)
+      document.removeEventListener("click", handleClick)
     }
+    // eslint-disable-next-line react-hooks/exhaustive-deps
```

Comment:
Why disable this?

Reply:
If we remove this, we get the following warning

<img width="1274" alt="image" src="https://user-images.githubusercontent.com/3513494/172621903-59be37f3-d71d-4fd2-8930-253040911eb4.png">

Reply:
Theoretically if we say, enabled this prop later on, this would break. Just as a general rule we should follow exhaustive-deps.

Reply:
> Theoretically if we say, enabled this prop later on, this would break

Yes, but it seems to me that few people will do so, and for this reason I added `eslint-disable-next-line`. But I have nothing against your suggestion and I will specify `openDropdownByClick` in `deps

Reply:
Done